app/api/share/route.ts

```diff
@@ -19,21 +19,17 @@ export const POST = middleware([sessionExists()], async (req, props) => {
 
     const base64data = Buffer.from(form).toString("base64");
 
-    const templateID = process.env.TEMPLATE_ID;
-    const notifyClient = getNotifyInstance();
-
     // Here is the documentation for the `sendEmail` function: https://docs.notifications.service.gov.uk/node.html#send-an-email
     await Promise.all(
       emails.map((email: string) => {
-        return notifyClient.sendEmail(templateID, email, {
-          personalisation: {
-            application_file: {
-              file: base64data,
-              filename: `${slugify(filename)}.json`,
-              sending_method: "attach",
-            },
-            subject: "Form shared | Formulaire partagé",
-            formResponse: `
+        return sendEmail(email, {
+          application_file: {
+            file: base64data,
+            filename: `${slugify(filename)}.json`,
+            sending_method: "attach",
+          },
+          subject: "Form shared | Formulaire partagé",
```

Comment:
Can't remember does the "real subject" come from the template itself?

Seeing the current email subject as `GC Forms | Formulaires GC`

Reply:
Not for the Template used throughout the app.  We have a personalization holder setup in the Subject of the Template.